scilpy/io/streamlines.py

```diff
@@ -23,3 +28,63 @@ def check_tracts_same_format(parser, tractogram_1, tractogram_2):
     if not ext_1 == ext_2:
         parser.error(
             'Input and output tractogram files must use the same format.')
+
+
+def load_trk_in_voxel_space(trk_file, anat=None, grid_res=None,
```

Comment:
to remove, use dipy.io.streamlines - load_tractogram and save_tractogram instead